lib/benchmark/driver/configuration.rb

```diff
@@ -25,6 +25,7 @@ def loop_count
   Executable = Struct.new(:name, :command) do
     def self.parse(name_path)
       name, path = name_path.split('::', 2)
+      path = File.expand_path(path)
```

Comment:
When path which does not include "::" is given (ex: `-e ruby`), `path` becomes `nil` and this would raise `TypeError`.